lib/src/test/java/io/ably/lib/test/realtime/RealtimeInitTest.java

```diff
@@ -94,7 +94,6 @@ public void init_host() {
 			ClientOptions opts = new ClientOptions(testVars.keys[0].keyStr);
 			opts.restHost = "some.other.host";
 			ably = new AblyRealtime(opts);
-			assertEquals("Unexpected host mismatch", Defaults.getHost(opts), opts.restHost);
```

Comment:
AblyRealtime is using Http#setHost https://github.com/ably/ably-java/pull/80/files#diff-8a8a56ece4e68a2d0fbeb6639c2e5469R136